apps/color-picker/src/types.ts

```diff
@@ -0,0 +1,15 @@
+export interface Color {
+  id: string;
+  name: string;
+  value: string;
+}
+
+export interface Theme {
+  id: string;
+  name: string;
+  colors: Color[];
+}
+
+export interface AppInstallationParameters {
+  themes: Theme[];
```

Comment:
We only use one theme at the moment. Should we still store an array here? Any plans on multiple themes?